lib/elixir/lib/gen_server.ex

```diff
@@ -748,7 +748,16 @@ defmodule GenServer do
   """
   @spec stop(server, reason :: term, timeout) :: :ok
   def stop(server, reason \\ :normal, timeout \\ :infinity) do
-    :gen.stop(server, reason, timeout)
+    case whereis(server) do
+      nil ->
+        exit(:noproc)
```

Comment:
Oops, this also has to be:

    exit({:noproc, {__MODULE__, :stop, [server, reason, timeout]}})